npdarrington/instaquizzes
# Nd testing app 2

### What’s this PR do?  
- Fixes the integration test between App and Question that broke due to react-router-dom being used in previous branch.
- Adds MemoryRouter around all instances of App rendering.
- Removes 13 questions in the mockQuizQuestions file to be left with only 2.
 
### Where should the reviewer start?  
- App.tsx, App.test.tsx, mockQuizQuestions.tsx
- Run `npm install` if you have not done so since `@types/react-router-dom` install.
 
### How should this be manually tested?  
- `npm test` to run the newest tests on App.test.tsx
- All test should be passing.
- Change expect assertions to ensure that it is not a false pass.
 
### Any background context you want to provide?  
- On the `should restart with a new game when Start New Game is clicked`, there is a `waitFor` that wraps the expect assertion to prevent an error where hooks is being called again, setting state mid test and throwing an error in console. This `waitFor` is prevent the `act` error test that prints inside of the console.
 
### What are the relevant tickets?  
- closes #5 
- closes #15 
- closes #17
